src/main.cpp

```diff
@@ -59,6 +59,7 @@ void print_help(const char *argv0) {
     << "    --no-tty         Surpress interactive build progress interface"              << std::endl
     << "    --no-wait        Do not wait to obtain database lock; fail immediately"      << std::endl
     << "    --no-workspace   Do not open a database or scan for sources files"           << std::endl
+    << "    --strict         Set PATH shell variable as empty (default is /usr/bin:/bin)"<< std::endl
```

Comment:
Do you envision this flag being used for other types of strictness that we may want to optionally enable in the future, or just the PATH variable? If so, I think this is fine as is, but if you think this would only ever be used for the PATH, then I'd probably prefer a more detailed option name like `--strict-path`.